### ISSUE TITLE
Using Lag-Llama for anomaly detection in multivariate time series data

### ISSUE THREAD

**Issue by thisthq** [time-series-foundation-models/lag-llama#42]
Hi all,

Thank you for open-sourcing this library.

I am working on the task of anomaly detection in multivariate time series data. I would like to know how to use Lag-Llama to accomplish this. Is it possible for Lag-Llama to achieve this through fine-tuning?

Thanks!

**Comment by ashok-arjun** [comment]
Hi, we have not tested the model for the anomaly detection task. Since it involves a different objective function (loss function), you would have to change the last layer to something more appropriate, and then finetune the model.

You may finetune just the last layer or the entire model.
